### ISSUE TITLE
Building from source fails with latest Qt

### ISSUE THREAD

**Issue by jcelerier** [clementine-player/Clementine#6088]
Qt 5.11 removed the qt5_add_module CMake macros (which were already deprecated since ... Qt 5.4 ?), which breaks the build.

**Comment by a17r** [comment]
This was fixed long before your post already: https://github.com/clementine-player/Clementine/commit/a4e362b9f76de7ef58e494479b159aa3a447c42c

And it is also obsolete again with Qt 5.11.1.

**Comment by EGD** [comment]
@a17r this commit not replace `qt5_add_module` macro in `3rdparty/libmygpo-qt5/src/CMakeLists.txt:89` file

**Comment by a17r** [comment]
That is true, but then I don't care about outdated bundled libs. This is fixed in `libmygpo-qt` git master, anyway.

**Comment by EGD** [comment]
It is probably time to merge master into `qt5` branch?

**Comment by a17r** [comment]
Imo the bundled lib should simply be dropped.

**Comment by jcelerier** [comment]
@a17r well, I did (at the time of my report) a fresh git clone at branch qt5 and had the problem.

**Comment by a17r** [comment]
Yes, I didn't think of that. Just install libmygpo-qt from your package manager.
